src/lib/apparmor/profiles.rb

```diff
@@ -208,7 +216,14 @@ def dialog_content
         VSpacing(0.3),
         # Footer buttons
         HBox(
-          HWeight(1, PushButton(Id(:changeMode), _('Change mode'))),
+          HWeight(1,
+            PushButton(Id(:setEnforce),
+              ((table_items.first.params[1] == "enforce") ? Opt(:disabled) : Opt()),
+              _("S&et to 'enforce'"))),
```

Comment:
Minor nitpicking: having the hotkey as `Set to '&enforce'` would be more in sync with `Set to '&complain'` (even if it technically doesn't change the hotkey, at least for english).